extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -376,8 +376,9 @@ export const AzureProvisionDialog: React.FC = () => {
     } --createLuisResource=${createLuisResource} --createLuisAuthoringResource=${createLuisAuthoringResource} --createCosmosDb=${createCosmosDb} --createStorage=${createStorage} --createAppInsights=${createAppInsights} --createQnAResource=${createQnAResource}`;
 
     const instructions = formatMessage(
-      'I am working on a Microsoft Bot Framework project, and I now require some Azure resources to be created.' +
-        ' Please follow the instructions below to create these resources and provide them to me.\n\n' +
+      'I am creating a conversational experience using Microsoft Bot Framework project. For my project to work,' +
```

Comment:
Why are we using multiple strings instead of just one long string? Readability?

Reply:
Yes for readability.  We could probably use a line continuation backslash but that can introduce whitespace issues.

Reply:
I think breaking them into sentences would be best

Reply:
broke into sentences 👍